metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -445,18 +464,91 @@ def get_container_urns_by_filter(
             logger.debug(f"yielding {x['entity']}")
             yield x["entity"]
 
+    def get_urns_by_platform(
```

Comment:
Let's call this get_urns_by_filter or something

Eventually I'd like to add env and some other filtering options here, so it'd be nice to keep it generic